src/components/iconinitials/iconinitials.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import "./iconinitials.scss";
+
+export type IconInitialsProps = React.HTMLAttributes<HTMLDivElement> & {
+  initials: string;
+  size?: string;
+  fontSize?: string;
+};
+
+/**
+ * IconInitials component. Used to display an icon to show initial letters.
+ * @param initials
+ * @param props
+ * @constructor
+ */
+export const Iconinitials: React.FC<IconInitialsProps> = ({
+  initials,
```

Comment:
Can we:

 - Use the full name as input string
 - Generate the initials 
 - Use the full name on a aria-label attribute

Reply:
I like this idea, we let the component generate the initials for us. And indeed aria!!!